src/leveled_sst.erl

```diff
@@ -2735,17 +2742,22 @@ split_lists(KVList1, SlotLists, N, PressMethod, IdxModDate) ->
 %% provided may include pointers to fetch more Keys/Values from the source
 %% file
 merge_lists(
-        KVList1, KVList2, LevelInfo, SSTOpts, IndexModDate, SaveTombCount) ->
+        KVList1, KVList2, {IsBase, L}, SSTOpts, IndexModDate, SaveTombCount) ->
     InitTombCount =
         case SaveTombCount of true -> 0; false -> not_counted end,
-    merge_lists(KVList1, KVList2,
-                LevelInfo,
-                [], null, 0,
-                SSTOpts#sst_options.max_sstslots,
-                SSTOpts#sst_options.press_method,
-                IndexModDate,
-                InitTombCount,
-                #build_timings{}).
+    BuildTimings = 
+        case IsBase or lists:member(L, ?LOG_BUILDTIMINGS_LEVELS) of
```

Comment:
if we talk efficiency:  use `orelse` instead of `or` here. No need to check membership if `IsBase` is `true`.
On the other hand checking member of singleton list is rather cheap.